content/modules.md

```diff
@@ -263,3 +263,61 @@ export class AppModule {}
 ```
 
 The [Dynamic modules](/fundamentals/dynamic-modules) chapter covers this topic in greater detail, and includes a [working example](https://github.com/nestjs/nest/tree/master/sample/25-dynamic-modules).
+
+#### Troubleshooting
+
+The most common problem when you are new to Nest is importing and exporting services to be used in other modules.
+
+Do not import services from modules directly into another module, instead import modules into other modules.
+
+To make this more clear, let's look at a simple example.
+
+Imagine we have `ProductModule` that contains `ProductService`, and a second module `OrderModule` containing the `OrderService`. `OrderService` is dependent on `ProductService` to create an order.
+
+If you import `ProductService` directly into the imports of `OrderModule` like this:
+```
+@Module({
+  imports: [
+    ProductService // This is incorrectly imported
+  ],
```

Comment:
in nestjs v9, if `ProductService` is annotated with `@Injectable()`, the app will crash

Reply:
As @micalevisk, in v9 you'll get a descriptive error that this doesn't make sense.